src_c/imageext.c

```diff
@@ -215,9 +215,14 @@ image_load_ext(PyObject *self, PyObject *arg)
     }
 
     if (surf == NULL){
-        SDL_ClearError();
-        PyErr_SetString(PyExc_FileNotFoundError, "No such file or directory.");
-        return NULL;
+        if (!strcmp(IMG_GetError(), "Couldn't open fname.asdf")){
```

Comment:
I think you should remove fname.asdf from your strcmp method. Basically you can only check
`if (!strcmp(IMG_GetError(), "Couldn't open")){`. In the comment illume left he meant that it returns filename.extension (fname.asdf was just the example) and then this should work.

Reply:
Thanks for the input. I ended up using strncmp() to compare only the beginning of IMG_GetError and I also add a check for PY3 because FileNotFoundError is not defined in Python2.7.
All the tests seem to be passing now. Please let me know if there was anything else to change.